### ISSUE TITLE
PvP oriented gameplay

### ISSUE THREAD

**Issue by ghost** [Daenks/Meridian59_103#308]
What does everyone think about creating more pvp oriented game play? 

Everything from scheduled tournaments to system generated events.  Currently all we have is the usual log on, run around the world and create conflict  if ur lucky enough to run into someone to pick a fight.

I think one thing this game has always been lacking is real pvp incentive.  The end game in general is pretty lacking for both pve and pvp.  I think if we start putting some energy into this area of the game, there wont be as many pull requests that target random pking n such.  

Tldr - we need end game content and pvp incentive.
